OCPBUGS#19618: Updated the reqs to install LVMS

sr1kar99 opened the pull request:
Version(s):
4.12+

Issue:
[OCPBUGS-19618](https://issues.redhat.com/browse/OCPBUGS-19618)

Link to docs preview:
[Preview](https://72691--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-deployment-requirements-for-sno-ran_logical-volume-manager-storage) 


QE review:
- [x] QE has approved this change.

sr1kar99 commented:
@mdargatz
Can you please review this PR?
Thanks.

maxisses commented:
> @mdargatz Can you please review this PR? Thanks.

Looks good to me!

sr1kar99 commented:
/label peer-review-needed

dfitzmau commented:
/remove-label peer-review-needed

/label peer-review-in-progress

dfitzmau commented on modules/deploying-lvms-on-sno-cluster.adoc:
```diff
@@ -30,17 +30,16 @@ You can deploy {lvms} using one of the following:
-* Every managed cluster has dedicated disks that are used to provision storage.
-
-Before deploying {lvms} in a private CI environment where you can reuse the virtual machines and storage devices, ensure that you have wiped the disks that are not in use. 
+Before deploying {lvms} in a private CI environment where you can reuse the storage devices that you configured in the previous {lvms} installation, ensure that you have wiped the disks that are not in use. If you do not wipe the disks before installing {lvms}, you cannot reuse the disks without manual intervention.
```
This clause places the important info at the end of the sentence:

> you cannot reuse the disks without manual intervention.

How about:

>, you must take manual steps to reuse the disks.

WDYT? The original frontloads with a "cannot" clause but then finished with a workaround tip.

sr1kar99 replied:
Yes. I think this improves clarity.
Updated as per the suggestion.

dfitzmau commented on modules/deploying-lvms-on-sno-cluster.adoc:
```diff
@@ -30,17 +30,16 @@ You can deploy {lvms} using one of the following:
 You cannot wipe the disks that are in use.
 ====
 
+If you want to install {lvms} by using {rh-rhacm-first}, ensure that you have installed {rh-rhacm} on an {product-title} cluster.
```
Might be obious, but you could state "See the Installing LVM Storage using RHACM section".

sr1kar99 replied:
Updated as per the suggestion.

dfitzmau commented on modules/deploying-lvms-on-sno-cluster.adoc:
```diff
@@ -30,17 +30,16 @@ You can deploy {lvms} using one of the following:
 ====
 
+If you want to install {lvms} by using {rh-rhacm-first}, ensure that you have installed {rh-rhacm} on an {product-title} cluster.
+
```
This plus is hidden, luckily, but sometimes it can rear it's nasty head in the published docs. I think it can be removed without causing any issues as no indentation is required for this intro block.

sr1kar99 replied:
Updated as per the suggestion.

dfitzmau commented:
Hi @sr1kar99 . Nice work. I added minor comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

sr1kar99 commented:
> Hi @sr1kar99 . Nice work. I added minor comments inline.
> 
> /remove-label peer-review-in-progress
> 
> /label peer-review-done

Hi @dfitzmau , Thank you for the review.
Updated the content as per the suggestions.
PTAL. 
Thanks.

dfitzmau commented:
> Hi @sr1kar99 . Nice work. I added minor comments inline.
> 
> /remove-label peer-review-in-progress
> 
> /label peer-review-done

Thanks, @sr1kar99 . LGTM. Feel free to move this PR to the next review cycle (QE maybe as the box is unticked).

sr1kar99 commented:
@radeore 
Can you please review this PR?
Thanks.

radeore commented:
/lgtm

sr1kar99 commented:
/label merge-review-needed

Srivaralakshmi commented:
Hi, @sr1kar99: Nice work! I see that this PR has technical updates. As per the [merge review checklist](https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.qeeeis1fmrjg), we need QE review and approval.

Please complete the QE review and place the PR merge request after you get approval from QE. Thanks!

sr1kar99 commented:
> Hi, @sr1kar99: Nice work! I see that this PR has technical updates. As per the [merge review checklist](https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.qeeeis1fmrjg), we need QE review and approval.
> 
> Please complete the QE review and place the PR merge request after you get approval from QE. Thanks!

Hi @Srivaralakshmi 
QE review for this PR is done. 
@radeore is the QE who reviewed this PR.
I had not updated QE review status in the PR description. Sorry about that.

Srivaralakshmi commented:
/cherrypick enterprise-4.12

Srivaralakshmi commented:
/cherrypick enterprise-4.13

Srivaralakshmi commented:
/cherrypick enterprise-4.14

Srivaralakshmi commented:
/cherrypick enterprise-4.16

sr1kar99 commented:
Hi @Srivaralakshmi 
Cherry picked to 4.12, 4.13, 4.14. Could you PTAL and merge the PRs.
Thanks.
